### PR TITLE
Añadir documentacion del calculo de TIs de las posiciones

### PR DESCRIPTION
**Pull request by Xevib**
### Description:

Añadir documentacion del calculo de TIs de las posiciones

### Tasks:

- [x]  Añadir texto
- [x] Traducir

### Links:

* [Posicions](http://builds.gisce.net/powerp-docs/powerp_imp_doc_pos_ti/ca/distri/posiciones/)
* [Posiciones](http://builds.gisce.net/powerp-docs/powerp_imp_doc_pos_ti/es/distri/posiciones/)

### CODE REVIEW

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_doc_pos_ti/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_doc_pos_ti/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_doc_pos_ti/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_doc_pos_ti/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_doc_pos_ti/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

File: docs/distri/posiciones.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#Posicions
+
+## Calcul de TI
+
+Per el calcul de TI s'usen els seguents parametres:
+
+- Tensió: S'usa la `Tensió` de la posició
```
Comment (jaumef) [inline docs/distri/posiciones.md]: CA: 
```suggestion
- Tensió: S'utilitza la `Tensió` de la posició
```
ES:
-Tensión: Se **usar** `Tensión` de la posición
+Tensión: Se **utiliza la** `Tensión` de la posición

File: docs/distri/posiciones.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#Posicions
+
+## Calcul de TI
+
+Per el calcul de TI s'usen els seguents parametres:
```
Comment (jaumef) [inline docs/distri/posiciones.md]:
```suggestion
Per al càlcul de TI s'utilitzen els següents paràmetres:
```

File: docs/distri/posiciones.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+- Tensió: S'usa la `Tensió` de la posició
+- Situacio:
```
Comment (jaumef) [inline docs/distri/posiciones.md]:
```suggestion
- Situació:
```

File: docs/distri/posiciones.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+    1. Si te assignat el `Parc` s'usa el camp Localització
```
Comment (jaumef) [inline docs/distri/posiciones.md]:
```suggestion
    1. Si té assignat el `Parc` s'utilitza el camp Localització 
```

File: docs/distri/posiciones.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+
+## Calcul de TI
+
+Per el calcul de TI s'usen els seguents parametres:
+
+- Tensió: S'usa la `Tensió` de la posició
+- Situacio:
+
+    1. Si te assignat el `Parc` s'usa el camp Localització 
+    2. Sino s'usa el camp Tipus del tipus de `Subtestacio`
```
Comment (jaumef) [inline docs/distri/posiciones.md]:
```suggestion
    2. Sino s'utilitza el camp `Tipus` del tipus de `Subtestacio`
```

File: docs/distri/posiciones.md [hunk 6]
```diff
@@ -0,0 +1,13 @@
+#Posicions
+
+## Calcul de TI
```
Comment (jaumef) [inline docs/distri/posiciones.md]:
```suggestion
## Càlcul de TI
```

File: docs/distri/posiciones.md [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+
+Per el calcul de TI s'usen els seguents parametres:
+
+- Tensió: S'usa la `Tensió` de la posició
+- Situacio:
+
+    1. Si te assignat el `Parc` s'usa el camp Localització 
+    2. Sino s'usa el camp Tipus del tipus de `Subtestacio`
+
+- Tipus: S'usa el camp `Tecnologia` de la posicio
```
Comment (jaumef) [inline docs/distri/posiciones.md]:
```suggestion
- Tipus: S'utilitza el camp `Tecnologia` de la posició
```

File: docs/distri/posiciones.md (hunk 5 above)
Comment (jaumef) [inline docs/distri/posiciones.md]: En ES: Parece que no has cerrado bien la clave de HTML del codigo porque la siguiente linia empieza en negrita. Revisa ambas.

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_doc_pos_ti/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_doc_pos_ti/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_doc_pos_ti/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_doc_pos_ti/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_doc_pos_ti/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_doc_pos_ti/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_doc_pos_ti/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_doc_pos_ti/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Pull request merged**